Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+@Library(['ableton-utils@0.1.0', 'codenarc@0.5.0']) _
```

Comment:
Let's use `groovylint`.

Reply:
Ah crap, I didn't even notice that. Thanks!